app/src/main/res/layout/document__fragment__edit.xml

```diff
@@ -60,19 +60,6 @@
             android:orientation="horizontal" />
     </HorizontalScrollView>
 
-    <TextView
```

Comment:
this now is replaced by/resulting in a IO-Error instead now, instead of this message. Right?